src/mlagility/analysis/status.py

```diff
@@ -71,6 +78,8 @@ def recursive_print(
 
 def print_invocation(
     model_info: ModelInfo,
+    build_name: str,
```

Comment:
Discussion (I don't have a strong feeling either way):

The per-model output is getting quite verbose between this change and #324. We should consider adding some kind of `--verbosity` option in the future. Thoughts on when the right time to do that would be, and what are the right verbosity settings?

Reply:
+1

Reply:
Agreed. Created a new issue #334